corehq/apps/registration/views.py

```diff
@@ -210,12 +225,14 @@ def page_context(self):
             'reg_form': RegisterWebUserForm(
                 initial=prefills,
                 show_number=self.ab_show_number,
+                show_persona=self.ab_show_persona,
             ),
             'reg_form_defaults': prefills,
             'hide_password_feedback': settings.ENABLE_DRACONIAN_SECURITY_FEATURES,
             'implement_password_obfuscation': settings.OBFUSCATE_PASSWORD_FOR_NIC_COMPLIANCE,
             'show_number': self.ab_show_number,
             'ab_show_number': self.ab_show_number_context,
+            'ab_persona_field': self.ab_persona_field.context,
```

Comment:
Just from a readability / copy-and-pasting code perspective, it's weird that this is `ab_persona_field.context` but the line above is `ab_show_number_context `